Education_Pathways/frontend/src/components/CourseDescription.js

```diff
@@ -17,10 +38,14 @@ class CourseDescriptionPage extends Component {
     super(props)
 
     this.state = {
-      course_code: "",
+      course_code: this.props.course_code,
```

Comment:
This field can be leave as blank because the state will be set up in the calling function.